### PR TITLE
Added online for correlations between mwpc and tttx

### PR DESCRIPTION
**Pull request by jose-luis-rs**
---

Checklist:

* [x] Rebased against `dev` branch
* [x] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [x] Followed the [pull request guidlines](https://opensource.creativecommons.org/contributing-code/pr-guidelines/) and the [Git workflow](https://github.com/AnarManafov/GitWorkflow/blob/master/GitWorkflow.markdown)
* [x] Followed the [seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

File: analysis/online/R3BMwpcvsTttxOnlineSpectra.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    void Reset_Histo();
+
+  private:
+    TClonesArray* fHitItemsMw = nullptr;
+    TClonesArray* fHitItemsTttx = nullptr;
+    R3BEventHeader* fHeader = nullptr;
+
+    int fNEvents = 0;
+    int fTpat = 0;
+    double fMinDist = 10.; // mm
```
Comment (ryotani) [inline analysis/online/R3BMwpcvsTttxOnlineSpectra.h:111]: At this moment, we are not sure about the accuracy of the alignment of the detector, so can the default value be much larger, or define a selector to disable the condition of the min-distance?

Comment (jose-luis-rs) [reply analysis/online/R3BMwpcvsTttxOnlineSpectra.h:111]: The method to change this distance is already implemented:
`inline void SetMinDist(double min) { fMinDist = min; }`

Comment (ryotani) [reply analysis/online/R3BMwpcvsTttxOnlineSpectra.h:111]: That's correct. Thank you!

**Comment by ryotani** [comment]
Sorry for asking you again, but just in case you have time to modify it... Can another set of histograms be added showing the correlations of hit multiplicities of MW1 and TTTX? That is like a correlation of nHitsMw against the number of hits of tttx (det==0). Otherwise, you can just merge this PR, then I will work on them afterwards.
Thank you!

**Comment by jose-luis-rs** [comment]
I have added 3 histograms to check the multiplicities:
- mw1 vs tttx all
- mw1 vs tttx1
- mw1 vs tttx2

**Comment by ryotani** [comment]
Thank you very much!

**Pull request merged**